Тестування гіпотез для середнього значення
Розглянемо тестування гіпотез для середнього значення на
прикладі.
9 листопада 1965 року в енергосистемі США сталася аварія. 30
мільйонів людей протягом 13 годин перебували без світла. Це
аварія
відома
як
Northeast
Blackout
https://en.wikipedia.org/wiki/Northeast_blackout_of_1965 Через 9
місяців (10 серпня 1966) в NY Times опубліковане дослідження,
яке
стверджувало,
що
значно
в
Нью
Йорку
зросла
народжуваність. Видання вважало причиною саме Northeast
Blackout. Давайте проаналізуємо кількість новонароджених у
перші два тижні серпня 1966 і визначимо, чи це значення
статистично відрізняється від звичайного рівня народжуваності
в Нью Йорку (середня кількість новонароджених на той час
складала 430 на добу).

Для цієї вибірки маємо:
𝑥 = 432.21,
s = 40.48
n = 14
Сформулюємо нульову гіпотезу: "Відключення електроенергії у
листопаді 1965 року впливу на кількість новонароджених не
має, тобто середнє значення таке ж, як і в інші місяці".

In [1]:
library(dplyr)


Attaching package: ‘dplyr’


The following objects are masked from ‘package:stats’:

    filter, lag


The following objects are masked from ‘package:base’:

    intersect, setdiff, setequal, union




In [3]:
# В R є вбудований функціонал для проведення тесту на значущість. Це функція t-test
newborns <- c(452, 470, 431, 448, 467, 377, 344, 449, 440, 457, 471, 463, 405, 377)

Використаємо функцію t-test, в якості параметрів вкажемо alternative="two.sided"
оскільки розглядаємо двосторонню альтернативу, mu=430 - значення середнього для
генеральної сукупності згідно нульової гіпотези та рівень
значущості conf.level = 0.95:

In [10]:
t.test(newborns, alternative = "two.sided", mu=430, conf.level = 0.95)


	One Sample t-test

data:  newborns
t = 0.20464, df = 13, p-value = 0.841
alternative hypothesis: true mean is not equal to 430
95 percent confidence interval:
 408.8384 455.5901
sample estimates:
mean of x 
 432.2143 


Це означає, що для кількості новонароджених у Нью Йорку p-
value = 0.814, що не дозволяє відкинути нульову гіпотезу для
рівня значущості 𝛼=0.05. Іншими словами, можна сказати що
різниця між нульовою гіпотезою та даними вибірки не є
статистично значущою. Тобто наші дані не підтверджують
гіпотезу, що рівень народжуваності у перші два тижні серпня
1966
відрізняється
від
звичного.
Тобто,
відсутність
електроенергії не мало впливу на рівень народжуваності.

Тестування гіпотез для пропорції
Розглянемо приклад тестування гіпотез для пропорції:
Чи відрізняється відсоток новонароджених хлопчиків від 50%?
У вибірці 200 новонароджених, з них 96 хлопчики.
𝐻0 : 𝑝 = 0.5
𝐻a : 𝑝 ≠ 0.5 Рівень довіри 𝛼 = 0.05

В R для тестування гіпотези про значення пропорції можна
використовувати
функцію
prop.test
з
параметрами
alternative = "two.sided" (може бути ще less або greater
для односторонніх альтернатив) та correct
=FALSE
(correct=TRUE означає застосування Yates' continuity correction
http://www.statisticshowto.com/what-is-the-yates-correction/)

In [17]:
prop.test(96, 200, alternative="two.sided", correct=FALSE)


	1-sample proportions test without continuity correction

data:  96 out of 200, null probability 0.5
X-squared = 0.32, df = 1, p-value = 0.5716
alternative hypothesis: true p is not equal to 0.5
95 percent confidence interval:
 0.4117917 0.5489621
sample estimates:
   p 
0.48 


Як бачимо, довірчий інтервал для рівня надійності 95%
[0.4117917, 0.5489621] включає значення 𝐻0
Тобто наші дані не підтверджують гіпотезу, що відсоток
хлопчиків серед новонароджених відрізняється від 50%.